pokepro/src/Pages/Pokemon/Pokemon.tsx

```diff
@@ -1,11 +1,13 @@
 // @ts-nocheck
-
+import { LazyLoadImage } from "react-lazy-load-image-component";
+import "react-lazy-load-image-component/src/effects/blur.css";
 import React, { useEffect, useState } from "react";
-import { useLocation } from "react-router-dom";
+import { Outlet, useLocation } from "react-router-dom";
 import { useParams } from "react-router-dom";
 import { useAppDispatch, useAppSelector } from "../../app/hook";
 import { getCurrentPokemonData } from "../../app/getCurrentPokemonData";
 import axios from "axios";
+import DetailsNavBar from "../../components/DetailsNavBar/DetailsNavBar";
 
 function Pokemon() {
```

Comment:
Also try to use type script where every its possible.

```suggestion
const PokemonDetailsPage: React.FC = () => {
```